### PR TITLE
igl | opengl | Fix index offset for IndexFormat::UInt8

### PR DESCRIPTION
**Pull request by vinsentli**

### CODE REVIEW

File: src/igl/opengl/RenderCommandEncoder.cpp [hunk 1]
```diff
@@ -55,6 +55,18 @@ int toGlType(IndexFormat format) {
   IGL_UNREACHABLE_RETURN(GL_UNSIGNED_INT)
 }
 
+uint8_t getIndexByteSize(int indexType) {
```
Comment (corporateshark) [inline src/igl/opengl/RenderCommandEncoder.cpp:58]: `GLenum`?

**Review by corporateshark — CHANGES_REQUESTED** [review]

File: src/igl/opengl/RenderCommandEncoder.cpp (hunk 1 above)
Comment (vinsentli) [reply src/igl/opengl/RenderCommandEncoder.cpp:58]: "RenderCommandEncoder::indexType_" is  int type, so i  keep it consistent with "indexType_".

Comment (vinsentli) [reply src/igl/opengl/RenderCommandEncoder.cpp:58]: If necessary, I can modify both indexType and RenderCommandEncoder::indexType_ together.

Comment (vinsentli) [reply src/igl/opengl/RenderCommandEncoder.cpp:58]: Use GLenum better

Comment (vinsentli) [reply src/igl/opengl/RenderCommandEncoder.cpp]: done

**Review by corporateshark — APPROVED** [review]

**Pull request closed without merging**